hybrid/SampleApps/NoteSync/NoteSync.xcodeproj/project.pbxproj

```diff
@@ -78,26 +78,33 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXContainerItemProxy section */
-		4FFEE60C1BFE914300B7AA8A /* PBXContainerItemProxy */ = {
+		829DA2711C1263010040F5F1 /* PBXContainerItemProxy */ = {
 			isa = PBXContainerItemProxy;
-			containerPortal = 4FFEE6051BFE914200B7AA8A /* SalesforceHybridSDK.xcodeproj */;
+			containerPortal = 829DA26A1C1263010040F5F1 /* SalesforceHybridSDK.xcodeproj */;
 			proxyType = 2;
-			remoteGlobalIDString = 82EFF2D016E7C57100A63CDF;
+			remoteGlobalIDString = CE4CE2E11C0E465B009F6029;
 			remoteInfo = SalesforceHybridSDK;
 		};
-		4FFEE60E1BFE914300B7AA8A /* PBXContainerItemProxy */ = {
+		829DA2731C1263010040F5F1 /* PBXContainerItemProxy */ = {
 			isa = PBXContainerItemProxy;
-			containerPortal = 4FFEE6051BFE914200B7AA8A /* SalesforceHybridSDK.xcodeproj */;
+			containerPortal = 829DA26A1C1263010040F5F1 /* SalesforceHybridSDK.xcodeproj */;
 			proxyType = 2;
 			remoteGlobalIDString = 82EFF2FF16E8049500A63CDF;
-			remoteInfo = HybridPluginTestApp;
+			remoteInfo = SalesforceHybridSDKTestApp;
 		};
-		4FFEE6101BFE914300B7AA8A /* PBXContainerItemProxy */ = {
+		829DA2751C1263010040F5F1 /* PBXContainerItemProxy */ = {
 			isa = PBXContainerItemProxy;
-			containerPortal = 4FFEE6051BFE914200B7AA8A /* SalesforceHybridSDK.xcodeproj */;
+			containerPortal = 829DA26A1C1263010040F5F1 /* SalesforceHybridSDK.xcodeproj */;
 			proxyType = 2;
 			remoteGlobalIDString = 8222729D19C39F4500FC537E;
-			remoteInfo = HybridPluginTestAppTests;
+			remoteInfo = SalesforceHybridSDKTestAppTests;
+		};
+		829DA2771C1263260040F5F1 /* PBXContainerItemProxy */ = {
```

Comment:
This is a common net-new addition to the projects that weren't building before.  Not sure why it wouldn't have been there before, but Xcode added it, so I'm leaving it there.